### PR TITLE
Update ICS 2024

### PR DESCRIPTION
**Pull request by duanhx1037**
### Which conference does this PR update?

- ICS 2024
### Related URL

- [https://ics2024.github.io/](https://ics2024.github.io/)

### CODE REVIEW

File: conference/DS/ics.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  dblp: ics
+  confs:
+    - year: 2024
+      id: ics24
+      link: https://ics2024.github.io/
+      timeline:
+        - abstract_deadline: '2024-01-12 19:59:59'
+          deadline: '2024-01-19 19:59:59'
+          comment: 'Registration deadline: Friday Jan 12, 2024, 11:59:59 AM UTC'
+      timezone: UTC+8
```
Comment (jacklightChen) [inline conference/DS/ics.yml:14]: should be AoE？

Comment (duanhx1037) [reply conference/DS/ics.yml:14]: You mean only AoE is supported here? I think UTC+8 is correct. From the hotcrp, 19:59 is our time(UTC+8), while the AoE is one day earlier (2024-01-11 and 2024-01-18).

Comment (Lukangkang123) [reply conference/DS/ics.yml:14]: > You mean only AoE is supported here? I think UTC+8 is correct. From the hotcrp, 19:59 is our time(UTC+8), while the AoE is one day earlier (2024-01-11 and 2024-01-18).

This project will automatically convert to Beijing time based on "timezone" and does not require manual conversion.

Comment (jacklightChen) [reply conference/DS/ics.yml:14]: The official deadline timezone for ICS 2024 is AoE, and this project will adjust the time based on the user's local timezone (may be UTC+8, +7 or any others), as explained by @Lukangkang123.
image

Comment (duanhx1037) [reply conference/DS/ics.yml]: > The official deadline timezone for ICS 2024 is AoE, and this project will adjust the time based on the user's local timezone (may be UTC+8, +7 or any others), as explained by @Lukangkang123. Therefore, the correct item may be sth. like [23:59:59 AoE]. image

OK，I will update it

File: conference/DS/ics.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+- title: ICS
+  description: ACM International Conference on Supercomputing
+  sub: DS
+  rank: B
+  dblp: ics
+  confs:
+    - year: 2024
+      id: ics24
+      link: https://ics2024.github.io/
+      timeline:
```
Comment (jacklightChen) [inline conference/DS/ics.yml:10]: Please check the format, each subitem in `timeline` must contain a `deadline` property, standalone `abstract_deadline` is not allowed.

**Comment by jacklightChen** [comment]
It seems that Mergify is not working, we have to wait for its recover = =

**Comment by jacklightChen** [comment]
@mergifyio refresh

**Review by jacklightChen — APPROVED** [review]

**Pull request merged**